code_comment/lib.py

```diff
@@ -104,6 +155,9 @@ def parse(self):
         def is_currently_multi_line_comment():
             return bool(tmp)
 
+        def is_python():
```

Comment:
i am not too much a fan of introducing a new method *just* specifically for python.

If anything, i'd prefer if the code below that uses it to simply reference:

```py
self.determine_code_language() == 'python'
```